package.json

```diff
@@ -105,6 +105,7 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.26.0",
+    "datatables.net-fixedheader": "^3.1.3",
```

Comment:
Can you expand on what this is for?

Reply:
Something I played with but couldn't get working. Removed.